ephios/core/views/shift.py

```diff
@@ -177,11 +177,11 @@ def setup(self, request, *args, **kwargs):
         super().setup(request, *args, **kwargs)
         self.object = self.get_object()
 
-    def delete(self, request, *args, **kwargs):
+    def form_valid(self, form):
```

Comment:
DeleteView is now a FormView.

Reply:
Didn't we have a complex delete view for qualifications? I think we should rework that one then.